cmake-modules/AddTestCMocka.cmake

```diff
@@ -77,7 +77,21 @@ function(ADD_CMOCKA_TEST _TARGET_NAME)
     endif()
 
     if(DEFINED ENV{VCPKG_ROOT} OR DEFINED ENV{VCPKG_INSTALLATION_ROOT})
-        set(CMOCKA_LIB ${CMOCKA_LIBRARIES})
+        if(DEFINED ENV{VCPKG_ROOT})
+            set(VCPKG_PATH $ENV{VCPKG_ROOT})
+        endif()
+        if(DEFINED ENV{VCPKG_INSTALLATION_ROOT})
+            set(VCPKG_PATH "$ENV{VCPKG_INSTALLATION_ROOT}/")
```

Comment:
Is the `/` at the end of the string intentional? Why don't we have the same thing above, in `set(VCPKG_PATH $ENV{VCPKG_ROOT})`

Also, is that why we surround it in quotes?

Reply:
it is intentional.
In the above lines, (usually Linux and Windows) the VCPK_ROOT includes the `/` at the end of the path
For VCPK_INSTALLATION_ROOT (Mac) the `/` is missing, that's why we add it.

the quotes are just required in the second one to make the concatenation. Above lines we don't need to